### PR TITLE
More screen updates elu

### PR DESCRIPTION
**Pull request by eric-lu-VT**
# More screen updates

App:
- Moved DBH into the main stage form
- Removed `height` variable from tree-census
   - Migration formally removes it from database
- Fixed plot stake alignment and coloring
- Removed tree tag autogeneration
   - Now initialized as blank
- Search feature on plots apparently got lost when resolving a past merge conflict; added it back in.
 
Retool:
- Adding tree species works
- Fixed delete species & delete label
- Deleted Ong Lab-requested plot censuses
- Fixed .csv export

## Type of Change

- [ ] New feature
- [x] Bug fix
- [ ] Breaking change
- [ ] Documentation update

## Referenced Issue(s)

- [172](https://github.com/dali-lab/inforest/issues/172)
- [169](https://github.com/dali-lab/inforest/issues/169)
- [184](https://github.com/dali-lab/inforest/issues/184)
- [186](https://github.com/dali-lab/inforest/issues/186)
- [175](https://github.com/dali-lab/inforest/issues/175)
- [149](https://github.com/dali-lab/inforest/issues/149)
- [183](https://github.com/dali-lab/inforest/issues/183)

## Todo
- Not sure if number keypad (without any other keys) is readily possible. Apparently iPad do not get the keyboard options available on phones. The best iPad can get is the regular keyboard opened to the symbols section opened, with the numbers across the top.
   - Could look into importing a custom keyboard, but this seems too much of a hassle for a very minor feature.

### CODE REVIEW

File: packages/server/src/db/migrations/20220126201103-create-trees-table.js [hunk 1]
```diff
@@ -30,9 +30,6 @@ module.exports = {
       dbh: {
         type: Sequelize.FLOAT,
       },
-      height: {
-        type: Sequelize.FLOAT,
-      },
```
Comment (eric-lu-VT) [inline packages/server/src/db/migrations/20220126201103-create-trees-table.js:35]: remove

File: packages/server/src/db/migrations/20220410191551-tree-and-census-entries.js [hunk 2]
```diff
@@ -30,9 +30,6 @@ module.exports = {
           dbh: {
             type: Sequelize.FLOAT,
           },
-          height: {
-            type: Sequelize.FLOAT,
-          },
```
Comment (eric-lu-VT) [inline packages/server/src/db/migrations/20220410191551-tree-and-census-entries.js:35]: remove

File: packages/server/src/db/migrations/20220410191551-tree-and-census-entries.js [hunk 3]
```diff
@@ -76,7 +73,6 @@ module.exports = {
               id: treeTagToTreeCensusId[existingTree.tag],
               treeTag: existingTree.tag,
               dbh: existingTree.dbh,
-              height: existingTree.height,
```
Comment (eric-lu-VT) [inline packages/server/src/db/migrations/20220410191551-tree-and-census-entries.js:79]: remove

File: packages/server/src/db/migrations/20220410191551-tree-and-census-entries.js [hunk 4]
```diff
@@ -95,7 +91,6 @@ module.exports = {
 
       // Remove columns from trees table.
       await queryInterface.removeColumn("trees", "dbh", { transaction });
-      await queryInterface.removeColumn("trees", "height", { transaction });
```
Comment (eric-lu-VT) [inline packages/server/src/db/migrations/20220410191551-tree-and-census-entries.js:98]: remove

File: packages/server/src/db/migrations/20220410191551-tree-and-census-entries.js [hunk 5]
```diff
@@ -264,7 +259,7 @@ module.exports = {
        *
        */
 
-      // Add trees.dbh, trees.height, trees.statusName, trees.tripId, trees.authorId.
```
Comment (eric-lu-VT) [inline packages/server/src/db/migrations/20220410191551-tree-and-census-entries.js:267]: remove

File: packages/server/src/db/migrations/20220410191551-tree-and-census-entries.js [hunk 6]
```diff
@@ -273,14 +268,6 @@ module.exports = {
...
         { transaction }
       );
-      await queryInterface.addColumn(
-        "trees",
-        "height",
-        {
-          type: Sequelize.FLOAT,
-        },
-        { transaction }
-      );
```
Comment (eric-lu-VT) [inline packages/server/src/db/migrations/20220410191551-tree-and-census-entries.js:283]: remove

File: packages/server/src/db/migrations/20220410191551-tree-and-census-entries.js [hunk 7]
```diff
@@ -308,13 +295,12 @@ module.exports = {
 
       Object.entries(mostRecentTreeCensusForTree).map(
         ([tag, mostRecentTreeCensus]) => {
-          const { dbh, height, tripId, authorId, updatedAt } =
```
Comment (eric-lu-VT) [inline packages/server/src/db/migrations/20220410191551-tree-and-census-entries.js:311]: remove

File: packages/server/src/db/migrations/20220410191551-tree-and-census-entries.js [hunk 8]
```diff
@@ -308,13 +295,12 @@ module.exports = {
...
+          const { dbh, tripId, authorId, updatedAt } =
             mostRecentTreeCensus;
           return queryInterface.bulkUpdate(
             "trees",
             {
               dbh,
-              height,
```
Comment (eric-lu-VT) [inline packages/server/src/db/migrations/20220410191551-tree-and-census-entries.js:317]: remove

**Comment by julian-george** [comment]
Remember to add new migration removing the height from the `treeCensus` model

**Review by julian-george — COMMENTED** [review]
Looks good!

**Review by julian-george — APPROVED** [review]

**Comment by julian-george** [comment]
This isn't great practice, but I added a bunch of UI tweaks to this branch that I've been meaning to add for a while, hence the few commits that I made. The main thing is that fields will now be the same height as their containers, so always be sure to set a static height for their containers. Everything before my commit looked good though, the only changes I made to your stuff was the removal of some console logs.

**Pull request merged**